src/config/env.ts

```
@@ -154,21 +154,72 @@ const s3ScanWebhookSecret = process.env.S3_SCAN_WEBHOOK_SECRET?.trim() || "chang
 if (parsed.data.NODE_ENV === "production") {
-  const missingFintechKeys: string[] = [];
-  if (!process.env.FLUTTERWAVE_SECRET_KEY) missingFintechKeys.push("FLUTTERWAVE_SECRET_KEY");
-  if (!process.env.FLUTTERWAVE_WEBHOOK_SECRET)
-    missingFintechKeys.push("FLUTTERWAVE_WEBHOOK_SECRET");
-  if (!process.env.PAYSTACK_SECRET_KEY) missingFintechKeys.push("PAYSTACK_SECRET_KEY");
-  if (!process.env.BILLS_WEBHOOK_SECRET) missingFintechKeys.push("BILLS_WEBHOOK_SECRET");
-  if (missingFintechKeys.length > 0) {
+  const invalidFintechKeys: string[] = [];
+  if (!process.env.FLUTTERWAVE_SECRET_KEY || isPlaceholderKey(process.env.FLUTTERWAVE_SECRET_KEY))
+    invalidFintechKeys.push("FLUTTERWAVE_SECRET_KEY");
+  if (!process.env.FLUTTERWAVE_WEBHOOK_SECRET || isPlaceholderKey(process.env.FLUTTERWAVE_WEBHOOK_SECRET))
```

ESLint

[error] Replace `!process.env.FLUTTERWAVE_WEBHOOK_SECRET·||·isPlaceholderKey(process.env.FLUTTERWAVE_WEBHOOK_SECRET)` with `⏎····!process.env.FLUTTERWAVE_WEBHOOK_SECRET·||⏎····isPlaceholderKey(process.env.FLUTTERWAVE_WEBHOOK_SECRET)⏎··`

(prettier/prettier)